python/test/unit/io/test_adios2.py

```diff
@@ -313,7 +313,15 @@ def test_vtx_reuse_mesh(tempdir, dim, simplex, reuse):
         1 if reuse else 3
     )  # For mesh variables the step count is 1 if reuse else number of writes
 
-    adios_file = adios2.open(str(filename), "r", comm=mesh.comm, engine_type="BP4")
+    # backwards compatibility adios2 < 2.10.0
```

Comment:
Could you make clear with a comment which part of the try/except clause is for v2.9 and which is for v2.10?